### ISSUE TITLE
New launcher icon needed

### ISSUE THREAD

**Issue by maxme** [wordpress-mobile/WordCamp-Android#13]
Current icon is good enough for a beta, but shouldn't be used in the public app:

https://github.com/wordpress-mobile/WordCamp-Android/blob/db22f89e6c33570ecd00cde3373c969be4ba60eb/app/src/main/res/mipmap-xxxhdpi/ic_launcher.png

**Comment by maxme** [comment]
ping @hugobaeta and @camworld

**Comment by hugobaeta** [comment]
Since the rebranding of WordCamp Central might be [a discussion that will take it's time](https://meta.trac.wordpress.org/ticket/1082), I'm proposing we make the app icon based on a dashicon already existing for camptix, the [Tickets Dashicon](https://developer.wordpress.org/resource/dashicons/#tickets). Here's a preview of what I'm proposing:

![App Icon Proposal](https://cldup.com/VInVZsQnZM.png)

If y'all agree on this, I can make a patch (if you tell me where in the app repo the icons are located).

cc/ @maxme @camworld

**Comment by markoheijnen** [comment]
I'm not really a fan of using a ticket like icon for the app. It feels that the app then is only for tickets.
That said, having another W logo also will not help so this could work.

One of my random thoughts was having the megaphone but then with a WordPress logo: https://developer.wordpress.org/resource/dashicons/#megaphone

**Comment by camworld** [comment]
No real opinion on this except that it shouldn't cause confusion. I wonder if we could use the W mark but incorporate the word 'WordCamp" into the icon to differentiate it from the WordPress app icon.

The full re-brand of WordCamp Central and the associated logo/icon(s) is going to take some time, as defined by Hugo.

**Comment by hugobaeta** [comment]
So, I iterated on this a bit further and here's what came of it:

![ic_launcher-proposals](https://cloud.githubusercontent.com/assets/1514660/8297278/9eaa226e-1953-11e5-907a-d0d38782fec7.png)
- **ic_launcher P1** - Quite similar to what I posted above. Further refined.
- **ic_launcher P2** - Some of Google's icons use the extra space in the icon file to break out of the primary shape (ie: Google Maps). I thought this was a fun idea to explore, and this is basically an extrapolation of the first icon.
- **ic_launcher P3** - Taking the idea further, using the app's accent color (yellow). My favorite <3

I have tested on my phone and currently have the **ic_launcher P3** option on my local repo ready to commit if we agree on it.

I zipped them all here, too: [ic_launcher P1](https://cloudup.com/c7OVJq33MKS), [ic_launcher P2](https://cloudup.com/cVTHzQ68N5d), [ic_launcher P3](https://cloudup.com/c8dcZxhSEHC)

**Comment by maxme** [comment]
> ic_launcher P1

IMO tickets look really too small on my device, I recognise the WordPress blue, but I can't tell it's a the WordPress W (perhaps I need a better pair of eyes).

> ic_launcher P2

My favorite

**Comment by maxme** [comment]
Note: Once we have the final design, I'll need a 512x512px version for the Play Store

**Comment by folletto** [comment]
P1 or P2 imho are the most interesting ones, mostly because I think that the color tone on P3 doesn't add much.

Brand wise they both work, while P3 feels to me a bit too far from the brand itself. Which could be a choice, but I don't feel it's worth it here.

One detail however is that in P2 and P3 the ticket below is perfectly (or almost perfectly) straight, which makes it a bit innatural and so it looks like the top one was a sticker torn off from the one below. If we go for P2 instead of P1 I'd suggest to make the two tickets both at some non-90° angle. :)

tl;dr
My preference right now is for P1, but could be P2 if the tickets were at a better angle. :)

**Comment by markoheijnen** [comment]
If we really have to go for the ticket, then I would go for P1. I agree what @folletto is saying.

**Comment by hugobaeta** [comment]
@folletto 

> One detail however is that in P2 and P3 the ticket below is perfectly (or almost perfectly) straight, which makes it a bit innatural and so it looks like the top one was a sticker torn off from the one below. If we go for P2 instead of P1 I'd suggest to make the two tickets both at some non-90° angle. :)

The original icon is like that, but I agree it needed some better movement. I iterated for a bit, and this is the result:

![ic_launcher_p4](https://cloud.githubusercontent.com/assets/1514660/8304890/debec24e-19a1-11e5-9017-842a8fe89cf3.png)

I get @maxme concerns about how small P1 is, so this is the one that works best IMO. @folletto any further feedback before I commit this one?

**Comment by folletto** [comment]
P4 looks great! :+1:

**Comment by hugobaeta** [comment]
@maxme ok if I commit this change to the repo?

**Comment by maxme** [comment]
@hugobaeta of course!

**Comment by aagam-shah** [comment]
Awesome logo @hugobaeta :+1:

**Comment by ratneshsonar** [comment]
How to solve this Error?

 Error retrieving parent for item: No resource found that matches the given name 'ThemeOverlay.AppCompat.Dark.ActionBar'.   styles.xml  /BaseActivity/res/values    line 22 Android AAPT Problem
